scripts/pi-hole/php/teleporter.php

```diff
@@ -174,6 +174,8 @@ function getWildcardListContent() {
 	header("Expires: 0");
 	if(ob_get_length() > 0) ob_end_clean();
 	readfile($archive_file_name);
+	ignore_user_abort(true);
+	unlink($archive_file_name);
```

Comment:
Does `readfile($archive_file_name);` block reliably so that we can be sure that the file is not unliked before it has finished sending to the user? Well, maybe it doesn't make a big difference, since unlinking a file is different than actually deleting it, as any open file pointer to this file should still be able to read it - just thinking aloud here, I guess.